src/OrbitGl/Batcher.cpp

```diff
@@ -10,6 +10,8 @@
 #include <math.h>
 #include <stddef.h>
 
+#include <memory>
```

Comment:
Because we're using smart pointers here. `memory` is transitively included, through `Batcher.h`, but we shouldn't rely on transitive includes (or do we have an exception for transitive includes from .h file?).